contrib/inventory/ec2.py

```diff
@@ -590,7 +590,7 @@ def get_rds_instances_by_region(self, region):
                 error = self.get_auth_error_message()
             if not e.reason == "Forbidden":
                 error = "Looks like AWS RDS is down:\n%s" % e.message
-            self.fail_with_error(error, 'getting RDS instances')
+                self.fail_with_error(error, 'getting RDS instances')
```

Comment:
No, that's the whole point of this PR.  That's what we had before. This will now error, unless the error is Forbidden, which I don't care about, and would rather just ignore.